libsubmit/providers/local/local.py

```diff
@@ -214,7 +163,7 @@ def submit(self, command, blocksize, job_name="parsl.auto"):
         script_path = "{0}/{1}.sh".format(self.script_dir, job_name)
         script_path = os.path.abspath(script_path)
 
-        wrap_command = self.launcher(command, taskBlocks=self.taskBlocks)
+        wrap_command = launchers.get(self.launcher, None)(command, self.tasks_per_node * self.nodes_per_block)
```

Comment:
If self.launcher is not set, launchers.get will return None which is not a function. We need to fix this.